scripts.js

```diff
@@ -0,0 +1,28 @@
+// console.log('hey')
```

Comment:
It looks like there are 2 `scripts.js` files: this one and the one in the `scripts` folder. Can one be deleted?